docs/troubleshooting.md

```diff
@@ -185,6 +180,30 @@ Logs provide context for what was happening when the issue occurred.
   vscode-powershell@microsoft.com. Please still open an issue though
   so we can track the work &mdash; other users may have the same issue.
 
+#### Provide Language Server Protocol payload logs
+
```

Comment:
Should we specify that this only applies to certain versions of the extension i.e. if you have 2019-10 or above?

Reply:
added!